scripts/main.js

```diff
@@ -131,20 +132,27 @@ if (userInput === "marketing") {
 } else if (userInput === "customer-service") {
     console.log(userInput + " is een leuke afdeling om te werken. Er werken op dit moment " + departments['customer-service'].numberOfEmployees + " medewerkers. ");
 } else {
-    console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.");
+    /*console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.");*/
+    document.getElementById('error-message').textContent = "Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.";
 }
 
 /*/!*Opdracht 4b*!/*/
 if (userInput === "marketing") {
 
     const userInput2 = prompt(
-        'Je koos '+[userInput] +'. Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in.\n' +
+        'Je koos ' + [userInput] + '. Over welke functie wil je meer weten? Voer een getal tussen 0 en 3 in.\n' +
         '0: ' + departments.marketing.jobs[0].title + ',\n' +
         '1: ' + departments.marketing.jobs[1].title + ',\n' +
         '2: ' + departments.marketing.jobs[2].title + ',\n' +
         '3: ' + departments.marketing.jobs[3].title);
     console.log(userInput2);
 
+    document.getElementById('role-title').textContent = departments[userInput].jobs[userInput2].title;
```

Comment:
wanneer een ongeldig nummer wordt ingevoerd, wordt er een foutmelding weergegeven in de console: `main.js:150 Uncaught TypeError: Cannot read properties of undefined (reading 'title')`. Hierdoor kom je nooit uit bij het else-block op regel 164 en wordt de error-message nooit weergegeven in de console. Dit kan je aanpassen door deze statements in het if-block (of switch als je het gaat aanpassen) te verwerken.